src/app/archive/components/archive-toolbar/archive-toolbar.component.html

```diff
@@ -1,38 +1,42 @@
 <div class="container mat-elevation-z8">
   <mat-form-field class="keywords" appearance="fill">
     <mat-label>Schlüsselwort-Suche</mat-label>
-    <mat-chip-list #chipList aria-label="Fruit selection">
+    <mat-chip-list #chipList>
       <mat-chip *ngFor="let keyword of filter.keywords" [selectable]="false" [removable]="true"
         (removed)="remove(keyword)">
         {{keyword}}
         <mat-icon matChipRemove>cancel</mat-icon>
       </mat-chip>
-      <input placeholder="..." [matChipInputFor]="chipList" [matChipInputSeparatorKeyCodes]="separatorKeysCodes"
+      <input [matChipInputFor]="chipList" [matChipInputSeparatorKeyCodes]="separatorKeysCodes"
         [matChipInputAddOnBlur]="true" (matChipInputTokenEnd)="add($event)">
     </mat-chip-list>
     <mat-icon matSuffix>search</mat-icon>
     <mat-hint>Durchsuche Inhalte und Tags</mat-hint>
   </mat-form-field>
-  <!-- <button mat-raised-button [matMenuTriggerFor]="minMenu">Mindestens: {{filter.minValue}}</button>
-  <button mat-raised-button [matMenuTriggerFor]="maxMenu">Höchstens: {{filter.maxValue}}</button> -->
-  <mat-form-field appearance="fill">
-    <mat-label>Einreichungsdatum</mat-label>
-    <mat-date-range-input [rangePicker]="picker">
-      <input matStartDate placeholder="Start date" [value]="filter.startDate" (dateChange)="onStartDateChanged($event)"
-        readonly>
-      <input matEndDate placeholder="End date" [value]="filter.endDate" (dateChange)="onEndDateChanged($event)"
-        readonly>
-    </mat-date-range-input>
-    <mat-datepicker-toggle matSuffix [for]="picker"></mat-datepicker-toggle>
-    <mat-date-range-picker #picker></mat-date-range-picker>
-  </mat-form-field>
   <button mat-icon-button [matMenuTriggerFor]="scoreMenu">
     <mat-icon>tune</mat-icon>
   </button>
 </div>
 
 <mat-menu class="slider-menu" #scoreMenu="matMenu" xPosition="before">
```

Comment:
I find it a bit unituitive, that the pop menu closes, even when I click inside the menu (e.g. after selecting a date).
I see two ways to resolve this issue:
1. We keep the menu open until the user clicks outside of the menu 
2. We introduce buttons "Filtern" und "Abbrechen", that the close the menu with and without filtering respectively.